crates/rpc/rpc-builder/src/lib.rs

```diff
@@ -1648,6 +1666,20 @@ impl WsHttpServerKind {
             let local_addr = server.local_addr()?;
             let server = WsHttpServerKind::WithCors(server);
             Ok((server, local_addr))
+        } else if let Some(secret) = auth_secret {
```

Comment:
this is now mutually exclusive with cors where cors takes precedence over jwt, which is probably not what we want here?

do we want to stack both or should jwt overule cors? 

I think we can just add another variant that stacks cors+auth?

Reply:
Will add a fourth variant for dealing with cors+auth

Reply:
or make jwt take precednce over cors?